{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "1d03de55_2500d4e8",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1000067
      },
      "writtenOn": "2020-11-27T23:32:44Z",
      "side": 1,
      "message": "It looks like I can\u0027t reproduce the error. Good.",
      "revId": "8a9938e3ce87589f8ef5b40bd4895455f0a30ce0",
      "serverId": "c360c8e5-7aa8-3905-a7a0-810934766dac"
    }
  ]
}